php-frontend/src/main/java/org/sonar/php/metrics/MetricsVisitor.java

```diff
@@ -105,7 +102,7 @@ private void setCounterMeasures() {
     fileMeasures.setStatementNumber(counter.getStatementNumber());
   }
 
-  private void setLineAndCommentMeasures(File file) {
+  private void setLineAndCommentMeasures(CompatibleInputFile file) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "file". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1172)